code/modules/events/crystal_event.dm

```diff
@@ -295,6 +300,27 @@ This section is for the crystal stabilizer item and the crystal from the closed
 	icon = 'icons/obj/stack_objects.dmi'
 	material_type = /datum/material/otherworld_crystal
 
+/*
+This section is for the signaler part of the crystal portals
+*/
+/obj/item/assembly/signaler/crystal_anomaly
+	name = "Nothing here"
+	desc = "Nothing to see here."
```

Comment:
Watch out, people with the popup examines will see this.

Reply:
what's the popup examines? and how it affect gameplay? this should not be seen anyway